src/httpQuery.ts

```diff
@@ -192,6 +193,7 @@ export function neon<
     readOnly: neonOptReadOnly,
     deferrable: neonOptDeferrable,
     authToken,
+    disableWarningInBrowsers,
```

Comment:
If the option is not specified here, do we inherit from the global configuration parameter? I think we probably should.

Reply:
I guess it makes sense. The implementation is a bit weird, because neonConfig in reality is a socket config that we don't use for the http client, but I guess this is good enough